hw/top_earlgrey/ip/alert_handler/top_earlgrey_alert_handler_reg.core

```diff
@@ -0,0 +1,20 @@
+CAPI=2:
```

Comment:
Added 'top_earlgrey' specific parameterizations of these sources. Had to put these in separate core files to ensure dependency order is maintained.